internal/runner/rspec.go

```diff
@@ -63,3 +65,21 @@ func (Rspec) discoveryPattern() DiscoveryPattern {
 		ExcludePattern: excludePattern,
 	}
 }
+
+// commandNameAndArgs returns the command name and arguments to run the Rspec tests
+func (Rspec) commandNameAndArgs(testCases []string, testCommand string) (string, []string, error) {
+	completedTestCommand := ""
+	if strings.Contains(testCommand, "{{testExamples}}") {
+		completedTestCommand = strings.ReplaceAll(testCommand, "{{testExamples}}", shellquote.Join(testCases...))
+	} else {
+		completedTestCommand = testCommand + " " + shellquote.Join(testCases...)
+	}
+
+	testCommandFields, err := shellquote.Split(completedTestCommand)
```

Comment:
Joining the command together to search for the string only to then split it again seems unnecessary and possibly still error-prone to me. One of the words from `shellquote.Split` should be `{{testExamples}}`, because it takes care of de-quoting each word, whether the user wrote the test command with `{{testExamples}}` or `'{{testExamples}}'` or `"{{testExamples}}"`.

What about:

```suggestion
	words, err := shellquote.Split(completedTestCommand)
	if err != nil {
		return "", []string{}, err
	}
	idx := slices.Index(words, "{{testExamples}}") 
	if idx < 0 {
		words = append(words, testCases...)
		return words[0], words[1:], nil		
	}
	words = slices.Replace(words, idx, idx+1, testCases...)
	return words[0], words[1:], nil	
```

Reply:
Good call! Joining the string and splitting the results does feel unnecessary and error-prone! 😅